MetroscopeModelingLibrary/Tests/Multifluid/HeatExchangers/CoolingTower_reverse.mo

```diff
@@ -107,7 +107,7 @@ equation
 
   // Observable for Calibration
   WaterOutletTemp_sensor.T_degC = WaterOutletTemp;
-  AirOutletTemp_sensor.T_degC = AirOutletTemp;
+  AirOutletTemp_sensor.T_degC = AirOutletTemp-10*time;
```

Comment:
Be careful to not forget `*time` in your model, it should be used only for testing